scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -137,13 +137,22 @@ <h4 class="mt-3">Case: {{case.display_name}}</h4><p class="text-muted {% if case
 
   </div>
 </div>
-{% if case.is_research %}
-  <div class="form-group">
-    <div class="btn-group">
-      <a class="btn btn-outline-dark" href="{{ url_for('variants.variants', institute_id=institute._id, case_name=case.display_name, variant_type='research') }}">Research SNV and INDELs</a>
-      <a class="btn btn-outline-dark" href="{{ url_for('variants.sv_variants', institute_id=institute._id, case_name=case.display_name, variant_type='research') }}">Research structural variants</a>
+{% if case.is_research%}
+  {% if case.track=="cancer" %}
+    <div class="form-group">
+      <div class="btn-group">
```

Comment:
This is also a little picky, but couldn't the divs here live outside the {%if%} clauses? :blush:

Reply:
Sure! I'll change it